### DISCUSSION TITLE
Db name and user name in logs for shared pgbouncer service

### DISCUSSION THREAD

**Discussion by arbansal01** [pgbouncer/pgbouncer#668] [Q&A]
In our scenario, we have 2 cluster

- Cluster will contain separate namespaces of each application. Each application will be pointed to same pgbouncer shared service
- Cluster will have pgbouncer namespace having separate conf for each application databases in pgbouncer.ini

Now, when we check he pgbouncer logs, there are statements where dbname and dbuser are coming as blank. As mentioned, pgbouncer will be shared by mutiple apps so it will be difficult for us to track that logs are coming for which dbs.

Attach is the sample log file. Please share how can we add dbname and dbuser in logs

![image](https://user-images.githubusercontent.com/60982521/148998808-b2bb3f43-e007-4a9a-9bd8-7c2e15bc4960.png)

**Comment by petere** [comment]
This would happen if the client disconnects before sending the user name or database name to pgbouncer. In your case, this might be a monitoring system checking the port but not sending anything.
